src/access/acp.test.ts

```diff
@@ -3653,3 +3686,250 @@ describe("internal_getActorAccessAll", () => {
     );
   });
 });
+
+describe("getGroupAccessAll", () => {
+  const groupAUrl = "https://some.pod/groups#groupA";
+  const groupBUrl = "https://some.pod/groups#groupB";
+  it("returns access set for any Group referenced in the ACR", () => {
+    const resourceWithAcr = mockResourceWithAcr(
+      "https://some.pod/resource",
+      "https://some.pod/resource?ext=acr",
+      {
+        policies: {
+          "https://some.pod/resource?ext=acr#policy": {
+            allOf: {
+              "https://some.pod/resource?ext=acr#applicable-allOf-rule": {
+                [acp.group]: [groupAUrl, groupBUrl],
+              },
+            },
+            allow: {
+              read: true,
+            },
+          },
+        },
+        memberPolicies: {},
+        acrPolicies: {},
+        memberAcrPolicies: {},
+      }
+    );
+
+    const access = internal_getGroupAccessAll(resourceWithAcr);
+
+    expect(access).toStrictEqual({
+      [groupAUrl]: {
+        read: true,
+      },
+      [groupBUrl]: {
+        read: true,
+      },
+    });
+  });
+
+  it("does not return access set for an agent", () => {
+    const resourceWithAcr = mockResourceWithAcr(
+      "https://some.pod/resource",
+      "https://some.pod/resource?ext=acr",
+      {
+        policies: {
+          "https://some.pod/resource?ext=acr#policy": {
+            allOf: {
+              "https://some.pod/resource?ext=acr#applicable-allOf-rule": {
+                [acp.agent]: ["https://some.pod/profile#agent"],
+              },
+            },
+            allow: {
+              read: true,
+            },
+          },
+        },
+        memberPolicies: {},
+        acrPolicies: {},
+        memberAcrPolicies: {},
+      }
+    );
+
+    const access = internal_getGroupAccessAll(resourceWithAcr);
+
+    expect(access).toStrictEqual({});
+  });
+
+  it("does not include access set for the public agent", () => {
```

Comment:
I don't think there's such a thing as "the public agent" - sounds like some kind of ombudsman? :P

```suggestion
  it("does not include access set for everyone", () => {
```

Reply:
Oops my bad, that sounds very influenced by [the ACP spec](https://github.com/solid/authorization-panel/blob/master/proposals/acp/definitions.md#access-rule), and it does look kinda bad ^^.